src/collective/collectionfilter/baseviews.py

```diff
@@ -2,11 +2,13 @@
 import json
 
 from plone import api
+from plone.memoize.view import memoize
 
 from collective.collectionfilter import PLONE_VERSION
 from collective.collectionfilter.filteritems import get_filter_items
+from collective.collectionfilter.interfaces import IGroupByCriteria
 from collective.collectionfilter.query import make_query
-from collective.collectionfilter.utils import base_query
+from collective.collectionfilter.utils import base_query, safe_iterable
```

Comment:
seems that something's wrong with `bin/code-analysis` ... this is not pep8 conform

Reply:
It's running and passing. E401 isn't in the ignore list.

Reply:
you're right ... pep8 says this is ok https://pep8.org/#imports